enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -378,7 +377,7 @@ func NewPebbleCache(env environment.Env, opts *Options) (*PebbleCache, error) {
 			if err := disk.EnsureDirectoryExists(blobDir); err != nil {
 				return err
 			}
-			pe, err := newPartitionEvictor(part, pc.fileStorer, blobDir, pc.leaser, pc.locker, pc.accesses, *opts.AtimeBufferSize, *opts.MinEvictionAge, opts.Name)
+			pe, err := newPartitionEvictor(part, pc.fileStorer, blobDir, pc.leaser, pc.locker, pc, pc.accesses, *opts.AtimeBufferSize, *opts.MinEvictionAge, opts.Name)
```

Comment:
nit: Don't feel strongly about this but would it be cleaner to just pass in the pebble cache as an argument, as opposed to passing it in as a versionGetter interface + pass in a bunch of its other fields individually?

Reply:
Yeah this bothered me too -- but pebble_cache holding evictors and evictors needing a reference to pebble cache seemed kinda gross. I'll prolly fix this in a followup.